src/xmipp/libraries/data/pdb.cpp

```diff
@@ -498,55 +510,79 @@ void PDBRichPhantom::read(const FileName &fnPDB, double pseudoatoms, double thre
         kind = line.substr(0,4);
         if (kind == "ATOM" || kind == "HETA")
         {
+			line.resize (80,' ');
+			i++;
```

Comment:
This change is not equivalent. Before you were increasing value of `i` every time the line was not empty. Now you do it only when `ATOM` or `HETA` are in the line. 
Is that correct?

Reply:
Probably that's actually better. We want `i` to be counting the atoms.

Reply:
This `i` is actually an artifact from something I was doing wrong with the pseudoatom parameter and should be deleted completely.